gax/src/main/java/com/google/api/gax/batching/v2/BatcherImpl.java

```diff
@@ -81,27 +88,36 @@ public BatcherImpl(
       BatchingDescriptor<ElementT, ElementResultT, RequestT, ResponseT> batchingDescriptor,
       UnaryCallable<RequestT, ResponseT> unaryCallable,
       RequestT prototype,
-      BatchingSettings batchingSettings) {
+      BatchingSettings batchingSettings,
```

Comment:
How will this affect the gax version? Will it get bumped to 2.0.0?

Reply:
The 2 implementations will live side by side. The old one will marked Deprecated. I don't think that we need a major version bump because existing batching code is marked as unstable

Reply:
sorry, what's the old implementation?

Reply:
old batching impl:
https://github.com/googleapis/gax-java/tree/master/gax/src/main/java/com/google/api/gax/batching

the new batching impl is here:
https://github.com/googleapis/gax-java/tree/master/gax/src/main/java/com/google/api/gax/batching/v2

This PR is just the next iteration on v2 impl